src/components/nav/index.js

```diff
@@ -29,12 +29,6 @@ const Nav = props => {
         <Link to="/about" className={styles.link}>
           About
         </Link>
-        <Link to="/nominate" className={styles.link}>
-          Nominate
-        </Link>
-        <Link to="/jobs" className={styles.link}>
-          Jobs
-        </Link>
```

Comment:
I did these as two separate commits to make it easier to revert either if we add these pages back in